src/main/java/com/synopsys/integration/detect/lifecycle/run/RunManager.java

```diff
@@ -409,4 +414,14 @@ private void runBlackDuckProduct(final ProductRunData productRunData, final Dete
         }
     }
 
+    private Set<String> collectCodeLocationNames(DetectableToolResult detectableToolResult, CodeLocationNameManager codeLocationNameManager, DirectoryManager directoryManager) {
```

Comment:
I don't like that this is called "collectCodeLocationNames" when it really "creates" code location names.

Reply:
that seems like a reasonable renaming